### PR TITLE
DonorProfiles domain namespace is now DonorDashboards

### LINKED ISSUE #5726: DonorProfiles domain namespace should be DonorDashboards
## Details

Currently, the Donor Dashboard feature domain namespace is DonorProfiles. It should be DonorDashboards.

## Acceptance Criteria
-   [x] References to DonorProfiles namespace have been replaced with DonorDashboards

### PR DESCRIPTION
**Pull request by henryholtgeerts**
**_NOTE: Changes in this PR are dependent upon changes in these PRs:_**
- https://github.com/impress-org/give-annual-receipts/pull/92
- https://github.com/impress-org/give-recurring/pull/1038

Resolves #5726 

## Description

This PR introduces changes to ensure that the DonorProfiles  domain namespace is instead referred to as DonorDashboards.

## Affects

This PR affects all references to the DonorProfiles namespace.

## Visuals

N/A

## Testing Instructions

1. Make sure you are testing with all the dependent PRs
2. Check various functionality of the DonorDashboard domain, to ensure that there are no glaring breaks

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [x] Relevant `@since` tags included in DocBlocks
-   [x] Changes logged to the `Unreleased` section of `CHANGELOG.md`
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: src/DonorDashboards/Tabs/EditProfileTab/Tab.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+namespace Give\DonorDashboards\Tabs\EditProfileTab;
+
+use Give\DonorDashboards\Tabs\Contracts\Tab as TabAbstract;
+use Give\DonorDashboards\Tabs\EditProfileTab\ProfileRoute;
+use Give\DonorDashboards\Tabs\EditProfileTab\LocationRoute;
+
+/**
+ * @since 2.10.0
+ */
+class Tab extends TabAbstract {
```
Comment (JasonTheAdams) [inline src/DonorDashboards/Tabs/EditProfileTab/Tab.php:12]: @henryholtgeerts What's the story with this introduction?

Comment (henryholtgeerts) [reply src/DonorDashboards/Tabs/EditProfileTab/Tab.php:12]: Hmm...not entirely sure why it thinks that's new...it exists in the same way on `develop`. Not a change I made as part of this PR.

Comment (henryholtgeerts) [reply src/DonorDashboards/Tabs/EditProfileTab/Tab.php:12]: Possibly auto-formatting by husky?

**Review by JasonTheAdams — APPROVED** [review]
Great job! Still works! 👍

**Pull request merged**